packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -20,11 +20,17 @@ import { generateImportsCode, Imports } from "./utils/generateImportsCode";
 
 type TsCodeRecordToStringObject = Record<string, string | number | undefined>;
 
-function tsCodeRecordToString(object: TsCodeRecordToStringObject) {
-    return `{${Object.entries(object)
+function tsCodeRecordToString(object: TsCodeRecordToStringObject, removeBrackets = false) {
```

Comment:
why this argument?

Reply:
Not all keys were defined inside the `tsCodeRecordToString` so the `{}` needed to be defined outside. 
I just moved everything inside `tsCodeRecordToString` now, so it's no longer needed.